platform-includes/getting-started-add-top-level-import/javascript.express.mdx

```diff
@@ -8,7 +8,8 @@ const app = express();
 
 // Add your routes, etc.
 
-// Add this after all routes and other middlewares are defined
+// Add this after all routes,
+// but before any and other middlewares are defined
```

Comment:
m: I think "middlewares" is a bit misleading as I'm fairly sure people also define middleware before they register their routes, no?

```suggestion
// but before any error handlers are defined
```

Reply:
Would "error-handling middleware" work?

Reply:
I'll go with this 👍